src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -45,6 +45,8 @@ public void send(PaymentCommand cmd) {
             );
             message.setLabel(cmd.getLabel());
 
+            LOG.info("Call doSend to send message payments queue");
```

Comment:
Is this one necessary?

Reply:
it is to see the reties.

Reply:
I see. How about something more high-level, like `About to send message to payments queue. Message ID: ...`?

Reply:
dosend logs these details

Reply:
I can change the description